pkg

```diff
@@ -163,7 +163,7 @@ Usage: $SELF [options] [projects]
   -d           Disable compiler optimizations for debugging.
   -f <file>    Use <file> as the docker-compose. Default: $COMPOSE_FILE
   -h           Print help message and exit
-  -L           Don't write logs to files - respects output levels on stderr/stdout as set by -q/-v
+  -L           Don$(echo \')t write logs to files - respects output levels on stderr/stdout as set by -q/-v
```

Comment:
does this have to be a subshell? Can it not just be escaped?

Reply:
Nope, if it's `\'`, then both characters get printed.

Both characters get printed if you do `''`, too.

Reply:
stunning